flagscale/train/models/deepseek_v3/deepseek_v3_model.py

```diff
@@ -103,23 +108,32 @@ def __init__(
             scatter_embedding_sequence_parallel=scatter_embedding_sequence_parallel,
             seq_len_interpolation_factor=seq_len_interpolation_factor,
         )
-        
-        if self.post_process and self.use_mtp_predictor:
+
+        if self.use_mtp_predictor and self.post_process:
```

Comment:
If pipeline parallelism is used, the `MTP` module is in the last pipeline stage, so the `self.post_process` is always `True.`